package/MDAnalysis/coordinates/GRO.py

```diff
@@ -200,7 +201,7 @@ def _read_first_frame(self):
             for pos, line in enumerate(grofile, start=1):
                 # 2 header lines, 1 box line at end
                 if pos == n_atoms:
-                    unitcell = np.float32(line.split())
+                    unitcell = np.float32(re.findall(r"(\d+\.\d{5})", line))
```

Comment:
I think this is probably a little strict, eg it won't parse "`1.0 1.0 1.0`" - There's probably people writing GRO files to less than 5dp precision.  I think something like: `(\d+\.?\d{0,5}\s*)` might work?

Reply:
I'm not an expert in regular expression, sorry. Your suggestion works. Thanks